web/src/components/auth/login/LoginForm.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { goTo } from '@/app/actions';
+import AuthService from '@/authentication/AuthService';
+import { handleAuthStep } from '@/authentication/helpers/handleAuthStep';
+import { Routes } from '@/models/routes/Routes';
+import { Center, Paper, Text } from '@mantine/core';
+import { Formik, FormikHelpers } from 'formik';
+import LoginFormView from './LoginFormView';
+import LoginFormValues, { defaultLoginValues } from './formValues';
+
+export const login = async (userAlias: string, password: string) => {
+    const authService = new AuthService();
+    const nextStepDetails = await authService.login({
```

Comment:
We'll need error handling, but we can handle that in a separate ticket.

Edit: You could handle the errors in onSubmit, actually. So maybe not so bad, but it should be improved.